set.mm

```diff
@@ -279590,27 +279595,27 @@ are univariate (or multivariate) polynomials. See Wikipedia "Polynomial
   Constant polynomial matrices
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 
-A _constant polynomial matrix_ is a polynomial matrix whose elements are
-constant polynomials, i.e. polynomials with no indeterminates. Constant
-polynomials are obtained by "lifting" a "scalar" (i.e. an element of the
-underlying ring) into the polynomial ring/algebra by a "scalar injection", i.e.
-applying the "algebra scalar injection function" ` algSc ` (see ~ df-ascl ) to
-a scalar ` A e. R `: ` ( ( algSc `` P ) `` A ) ` . In an analogous way,
-constant polynomial matrices (over the ring ` R `) are obtained by "lifting"
-matrices over the ring ` R ` by the function ` matToPolyMat ` (see
-~ df-mat2pmat ), called "matrix transformation" in the following.
+  A _constant polynomial matrix_ is a polynomial matrix whose elements are
+  constant polynomials, i.e., polynomials with no indeterminates.  Constant
+  polynomials are obtained by "lifting" a "scalar" (i.e. an element of the
+  underlying ring) into the polynomial ring/algebra by a "scalar injection",
+  i.e., applying the "algebra scalar injection function" ` algSc ` (see
+  ~ df-ascl ) to a scalar ` A e. R `: ` ( ( algSc `` P ) `` A ) ` .
+  Analogously, constant polynomial matrices (over the ring ` R `) are obtained
+  by "lifting" matrices over the ring ` R ` by the function ` matToPolyMat `
+  (see ~ df-mat2pmat ), called "matrix transformation" in the following.
 
-In this section it is shown that the set ` S = ( N ConstPolyMat R ) ` of
-constant polynomial ` N ` x ` N ` matrices over the ring ` R ` is a subring of
-the ring of polynomial ` N ` x ` N ` matrices over the ring ` R `
-( ~ cpmatsrgpmat ) and that ` T = ( N matToPolyMat R ) ` is a ring isomorphism
-between the ring of matrices over a ring ` R ` and the ring of constant
-polynomial matrices over the ring ` R ` (see ~ m2cpmrngiso ).  By this, it is
-shown that the ring of matrices over a commutative ring is isomorphic to the
-ring of scalar matrices over the same ring, see ~ matcpmric .  Finally
-` I = ( N cPolyMatToMat R ) ` , the transformation of a constant polynomial
-matrix into a matrix, is the inverse function of the matrix transformation
-` T = ( N matToPolyMat R ) ` , see ~ m2cpminv .
+  In this section it is shown that the set ` S = ( N ConstPolyMat R ) ` of
+  constant polynomial ` N ` x ` N ` matrices over the ring ` R ` is a subring
+  of the ring of polynomial ` N ` x ` N ` matrices over the ring ` R `
+  ( ~ cpmatsrgpmat ) and that ` T = ( N matToPolyMat R ) ` is a ring
+  isomorphism from the ring of matrices over a ring ` R ` onto the ring of
```

Comment:
I think it is usual to say "an isomorphism/a bijection between to classes"

Reply:
Yes, it's not an uncommon abuse ! But why be imprecise if it does not save any word ? Most often (that is, if $A \neq B$), a bijection from A to B is not the same thing as a bijection from B to A.  But the expression "between A and B" is symmetric in A and B (this is the same thing as "between B and A").

Reply:
In principle,  I would like to keep the expression "bijection/isomorphism **between**". Of course the reverse/inverse function ist not the same, but the two sets related by a bijection are regarded as equal or equivalent in a certain context. It is also commonly used in literatue, see, for example,
* J. Adamek "The: "... since there is a bijective correspondence between the class of objects and the class of identity morphisms ..."
* S. Lang "Algebra": "The existence of an isomorphism between two groups G and G'..."

In N. Bourbaki "Algebra 1", however, it's "an isomorphism of E onto E'" (maybe you refer to that).

In set.mm, we have 41 times "bijection between" and 12 times "isomorphism between"...

Reply:
> J. Adamek "The: "... since there is a bijective correspondence between the class of objects and the class of identity morphisms ..."
> S. Lang "Algebra": "The existence of an isomorphism between two groups G and G'..."

They are correct:
* "there is a bijective correspondence between A and B" is symmetric in A and B,
* "The existence of an isomorphism between two groups G and G'..." is symmetric in G and G',

but on the other hand,
* "The morphism f is an isomorphism from A to B." is **not** symmetric in A and B.

There is a subtle but important difference between the **existence** of an object with given properties and the **data** of such an object.

Reply:
OK, I accept your change.